### PR TITLE
Add missing dependency

### PR DESCRIPTION
**Pull request by mitogh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "gulp-markdox": "^0.1.1",
     "gulp-phpcs": "^1.1.1",
     "gulp-rename": "^1.2.2",
+    "gulp-sass": "^2.3.2",
```
Comment (adamf321) [inline package.json]: Per bb convo, the error is in the gulpfile, not here.

**Comment by mitogh** [comment]
Changes has been updated!

**Comment by adamf321** [comment]
@mitogh is this still required?

**Comment by mitogh** [comment]
Not really, it can either be closed or merged since the dependencies and gulp is going to be updated once we have other tasks completed.

**Pull request closed without merging**